src/axes/timeAxis.ts

```diff
@@ -491,7 +492,7 @@ export module Axes {
     }
 
     private _hideOverflowingTiers() {
-      let availableHeight = this.height();
+      let availableHeight = this.height() - this.margin();
```

Comment:
Is the margin intended to be a "soft" margin?  Note that the current implementation has the timeAxis not consider the margin at all